README.md

```diff
@@ -53,6 +53,22 @@ generate a key, with optional seed.
 curve defaults to `ed25519` (and no other type is currently supported)
 seed should be a 32 byte buffer.
 
+Output is in the form
+
+    {
+      "curve": "ed25519",
+      "public": "<public_key_here>.ed25519",
+      "private": "<private_key_here>.ed25519",
+      "id": "@<public_key_here>"
+    }
```

Comment:
recommend changing this to: 
```
    {
      "curve": "ed25519",  // this is the current curve we're using
      "public": "<public_key_here>.<curve_here>
      "private": "<private_key_here>.<curve_here>",
      "id": "@<public_key_here>.<curve_here>"
    }
```

actually the naming of those variables is getting a bit tricky. I think a concrete example would be much clearer. I recommend a real version of something like:

```js
    {
      "curve": "ed25519",  // our current curve
      "public": "abc123.ed25519",    // format: key.curve
      "private": "ee23412212323.ed25519",  // format: key.curve
      "id": "@abc123.ed25519"  // format: @public
    }
```

Reply:
I've changed it to use a real keypair. (Not one that belongs to anyone!)